### ISSUE TITLE
Android 1.0.19: cannot move cursor in alt text fields

### ISSUE THREAD

**Issue by ghost** [pixelfed/mobile-app#101]
It seems that in v1.0.19 on Andoid I cannot move the curesor in the alt text fields, i.e. to correct typos or do cut/copy/paste. I always have to delete everything back up to the typo and then retype everything.

**Comment by glitch2500** [comment]
It looks like it happens when the keyboard comes up.. if I minimize the keyboard, I'm able to move around, although doing so immediately brings the keyboard back up, and again I lose the ability to move around/highlight

**Comment by cisene** [comment]
Can confirm this behavior @glitch2500 mentions - not possible to move cursor when keyboard is visible, minimizing it allows cursor to be placed anywhere. Alt-text input box, under label "Set Alt Text". Version 1.0.0.23
